src/bitcoinrpc/bitcoin_rpc.py

```diff
@@ -106,7 +94,7 @@ async def acall(
         Pass keyword arguments to directly modify the constructed request -
             see `httpx.Request`.
         """
-        req = self.client.post(
+        req = await self.client.post(
```

Comment:
Since `post` method is already awaited, this should be named `response` and `resp` below should be renamed to `content`.